CHANGELOG.md

```diff
@@ -19,6 +19,7 @@
 
 ### Bug Fixes
 - Fix issue faced in the Ansible based operators when `jmespath` queries are used because it was not installed. ([#2252](https://github.com/operator-framework/operator-sdk/pull/2252))
+- Fix scorecard basic test when run with `olm-deployed` option set to true ([#2274](https://github.com/operator-framework/operator-sdk/pull/2274))
```

Comment:
```suggestion
- Fix error `unrecognized install strategy` which is shown when the scorecard basic test is configured with `olm-deployed: true`. ([#2274](https://github.com/operator-framework/operator-sdk/pull/2274))
```

wdyt?

Reply:
Something more like:
```
Fix scorecard behavior such that a CSV file is read correctly when `olm-deployed` is set to `true`.
```

Reply:
any preference ? I prefer estroz's one. Shall I commit that ?

Reply:
It can be the @estroz's one. 👍

Reply:
Fixed